src/views/Insurance.vue

```diff
@@ -46,7 +46,7 @@ export default {
           path: '/insurance/mypolicies'
```

Comment:
I would suggest navigating by name, not by path as it's more difficult ot know to which component are we navigating to and there's bigger chance that path will be refactored in the future.